src/test/java/org/folio/gobi/MappingTest.java

```diff
@@ -176,25 +149,12 @@ void testSuccessLookupMappingOrdersPOListedPrintMonographWithNewAddedLookupsModG
     CompositePoLine pol = compPO.getCompositePoLines().get(0);
     //Then
     assertThat(pol.getOrderFormat(), is(CompositePoLine.OrderFormat.PHYSICAL_RESOURCE));
-    assertThat(compPO.getPoNumberSuffix(), equalTo(sufId));
```

Comment:
I suppose we shoud leave these assertions, and update lookupService with proper mockings. As well as in other places

Reply:
fixed